src/mock/productDetail.ts

```diff
@@ -0,0 +1,17 @@
+import mainImages from '../assets/images/detail-small.jpg';
+import secondaryImages from '../assets/images/small-t-shirt.jpg';
+import { Product } from '../types/product';
+
+export const productDetails: Product = {
```

Comment:
`productDetails` -> `product`

Reply:
Fixed

Reply:
@thanhtrantruong47 Please recheck the convention. Use MACRO_CASE for constant